roverrobotics_input_manager/scripts/ps4_mapper.py

```diff
@@ -24,31 +24,41 @@ def __init__(self):
         self._inputs = rospy.get_param('~inputs')
         self._scales = rospy.get_param('~scales')
         self.buttonpressed = False
+        self.togglebuttonpressed = False
+        self.counter2 = 0
         self.counter = 0
         self._attrs = []
         for attr in Status.__slots__:
             if attr.startswith('axis_') or attr.startswith('button_'):
                 self._attrs.append(attr)
-
-        self._pub = rospy.Publisher('cmd_vel/joystick', self._cls, queue_size=1)
-        self._pub_squ = rospy.Publisher('/joystick/square', Bool, queue_size=1, latch=True)
-        self._pub_triangle = rospy.Publisher('/joystick/triangle', Bool, queue_size=1, latch=True)
-        self._pub_circle = rospy.Publisher('/soft_estop/reset', Bool, queue_size=1)  # , latch =True)
-        self._pub_cross = rospy.Publisher('/soft_estop/trigger', Bool, queue_size=1)  # , latch =True)
-        self._pub_trim = rospy.Publisher('/trim_increment', Float32, queue_size=1)
+        self._feedback = Feedback()
+        self._pub = rospy.Publisher(
+            'cmd_vel/joystick', self._cls, queue_size=1)
+        self._pub_squ = rospy.Publisher('/mode_toggle', Bool, queue_size=1)
+        self._pub_triangle = rospy.Publisher(
+            '/joystick/triangle', Bool, queue_size=1, latch=True)
+        self._pub_circle = rospy.Publisher(
+            '/soft_estop/reset', Bool, queue_size=1)  # , latch =True)
+        self._pub_cross = rospy.Publisher(
+            '/soft_estop/trigger', Bool, queue_size=1)  # , latch =True)
+        self._pub_trim = rospy.Publisher(
+            '/trim_increment', Float32, queue_size=1)
         self._trim_incre_value = rospy.get_param('~trim_increment_value', 0.01)
```

Comment:
this type of value should be defined in ALL_CAPS in .py files, preferably at the top of the .py file